src/standalone/Soil/soil_albedo.jl

```diff
@@ -0,0 +1,200 @@
+    PAR_albedo_wet,
+    NIR_albedo_wet,
+    albedo_calc_top_thickness = FT(0.02),
+) where {FT}
+    return CLMTwoBandSoilAlbedo{FT, typeof(PAR_albedo_dry)}(
+        PAR_albedo_dry,
+        NIR_albedo_dry,
+        PAR_albedo_wet,
+        NIR_albedo_wet,
+        albedo_calc_top_thickness,
+    )
+end
+
+
+"""
+    albedo_from_moisture(θ_sfc::FT, albedo_dry::FT, albedo_wet::FT)
+
+Calculates pointwise albedo for any band as a function of soil surface moisture given
+the dry and wet albedo values for that band.
+
+CLM reference: Lawrence, P.J., and Chase, T.N. 2007. Representing a MODIS consistent land surface in the Community Land Model
+(CLM 3.0). J. Geophys. Res. 112:G01023. DOI:10.1029/2006JG000168.
+"""
+function albedo_from_moisture(
+    θ_sfc::FT,
+    albedo_dry::FT,
+    albedo_wet::FT,
+) where {FT}
+    Δ = max(FT(0.11) - FT(0.4) * θ_sfc, FT(0))
+    return min(albedo_wet + Δ, albedo_dry)
+end
+
+
+"""
+    update_albedo!(bc::AtmosDrivenFluxBC, albedo::CLMTwoBandSoilAlbedo, p, soil_domain, model_parameters)
+
+Calculates and updates PAR and NIR albedo as a function of volumetric soil water content at
+the top of the soil. If the soil layers are larger than the specified `albedo_calc_top_thickness`,
+the water content of the top layer is used in the calclulation. .The dry and wet
+albedo values come from a global soil color map and soil color to albedo map from CLM.
+
+CLM reference: Lawrence, P.J., and Chase, T.N. 2007. Representing a MODIS consistent land surface in the Community Land Model
+(CLM 3.0). J. Geophys. Res. 112:G01023. DOI:10.1029/2006JG000168.
+"""
+function update_albedo!(
+    bc::AtmosDrivenFluxBC,
+    albedo::CLMTwoBandSoilAlbedo,
+    p,
+    soil_domain,
+    model_parameters,
+)
+    (;
+        PAR_albedo_dry,
+        NIR_albedo_dry,
+        PAR_albedo_wet,
+        NIR_albedo_wet,
+        albedo_calc_top_thickness,
+    ) = albedo
+    FT = eltype(soil_domain.fields.Δz_top)
+    # checks if there is at least 1 layer centered within the top soil depth
+    if soil_domain.fields.Δz_min < albedo_calc_top_thickness
+        # We compute ∫H_θ_dz / N, where N =∫H_dz
+        # is a normalization,
+        # and then computing ∫[(H θ)/N]dz.
+        # N is integral of 1 from (surface-albedo_calc_top_thickness) to surface
+        N = p.soil.sfc_scratch
+        # zero all centers lower than boundary, set everything above to one
+        @. p.soil.sub_sfc_scratch = ClimaLand.heaviside(
+            albedo_calc_top_thickness + sqrt(eps(FT)),
+            soil_domain.fields.z_sfc - soil_domain.fields.z,
+        )
+        ClimaCore.Operators.column_integral_definite!(N, p.soil.sub_sfc_scratch)
+        # zeros all effective saturation at levels centered lower than boundary
+        @. p.soil.sub_sfc_scratch =
+            ClimaLand.heaviside(
+                albedo_calc_top_thickness + sqrt(eps(FT)),
+                soil_domain.fields.z_sfc - soil_domain.fields.z,
+            ) * p.soil.θ_l / N
```

Comment:
I see that in main we call `effective_saturation` here. I see that now you're dividing by `N` to normalize, but should this be `effective_saturation(ν, p.soil.θ_l, θ_r) / N`?

Reply:
I was wondering if I should make our parameterization exactly match CLM, in which case it would use theta_l, and not effective saturation(theta_l). I think it makes more sense to match CLM completely, because the parameters were calibrated using their scheme. But this is a separate change and I can make it in another PR?

Reply:
I think it makes sense to make that change in a separate PR. So continue using `effective_saturation` here, and later switch to theta_l